### PR TITLE
Add 'Dual-boot linux mint cloudready' post

### PR DESCRIPTION
**Pull request by likethemammal**

### CODE REVIEW

File: _posts/2016-04-26-cloudready-linux-mint.markdown [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: Dual-Boot Cloudready and Linux Mint
+tags: linux-mint cloudready dual-boot grub
+username: likethemammal
+---
```
Comment (likethemammal) [inline _posts/2016-04-26-cloudready-linux-mint.markdown]: need to add cross post to Razer blade post when it goes live

File: _posts/2016-04-26-cloudready-linux-mint.markdown [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+<center>*Source: Neverware*</center>
+
+## Prologue
+
+My girlfriend has been on an HP Chromebook 14 for while now. I was originally the one to recommend it and she's been generally satisfied expect for a few issues. One is that, even though there are many web based alternatives to common software, the Chromebook hardware isn't actually capable of running them for very long. Audio editing, flash gaming, and sometimes even regular usage causes scrolling, typing, and the like, to slug to a stutter.
+
+For a while I've wanted to setup one of my old machines for her to use. The machine would have to be as easy to use as Chrome OS, but capable of running traditional PC games and audio editing software (she's a musician).
+
+As I detailed in [my last post](), I recently got a Razer Blade gaming laptop setup as my personal machine. That freed up the laptop I had been using, an HP Probook 430 G1, to become hers.
```
Comment (likethemammal) [inline _posts/2016-04-26-cloudready-linux-mint.markdown]: Add link to razer blade post once it goes live

File: _posts/2016-04-26-cloudready-linux-mint.markdown (hunk 1 above)
Comment (akenn) [inline _posts/2016-04-26-cloudready-linux-mint.markdown]: Crossposts are used for referencing the same post on a different site. I don't have a problem using it for multiple things like this though.

Comment (likethemammal) [inline _posts/2016-04-26-cloudready-linux-mint.markdown]: oh didnt know that. i thought it was for a post referencing another post, nvm then

File: _posts/2016-04-26-cloudready-linux-mint.markdown [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+username: likethemammal
+---
+
+![Cloudready Logo](https://p5.zdassets.com/hc/settings_assets/187734/200050403/SZGZzvDCxuqvTTwMdcXTrw-Screen_Shot_2015-09-18_at_9.30.01_AM.png)
+
+<center>*Source: Neverware*</center>
+
+## Prologue
+
+My girlfriend has been on an HP Chromebook 14 for while now. I was originally the one to recommend it and she's been generally satisfied expect for a few issues. One is that, even though there are many web based alternatives to common software, the Chromebook hardware isn't actually capable of running them for very long. Audio editing, flash gaming, and sometimes even regular usage causes scrolling, typing, and the like, to slug to a stutter.
```
Comment (mediocregopher) [inline _posts/2016-04-26-cloudready-linux-mint.markdown]: `for a* while now`

Comment (mediocregopher) [inline _posts/2016-04-26-cloudready-linux-mint.markdown]: `except* for a few issues`

File: _posts/2016-04-26-cloudready-linux-mint.markdown [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+Neverware's focus is on revamping schools' old computer labs. Since a lot of that hardware is still better than today's Chromebooks, it runs Chromium beautifully. They also have a setup process to preserve the current Windows install and "dual boot" that with Cloudready.
+
+I use quotes around "dual-boot" because, when running the dual-boot setup, Cloudready is actually installed **into** the Windows partition, instead of along side it, completely taking it over. Its own boot software then manages whether it boots the user's Windows install or Chromium.
+
+I've provided diagrams below to explain the difference between this and a traditional dual-boot setup. We're aiming for a traditional dual-boot setup with Cloudready and Linux Mint in parallel. We may even try for the triple-boot!
+
+
+![Linux Mint and Cloudready boot structures](http://i.imgur.com/mK8mdFu.png)
```
Comment (mediocregopher) [inline _posts/2016-04-26-cloudready-linux-mint.markdown]: you can host the image in this repo, and link to it using the relative path. I did that in one of my previous posts, I think the terrain generation one. Then we're not hotlinking to somewhere else for image hosting,

File: _posts/2016-04-26-cloudready-linux-mint.markdown [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+#####Get Linux Mint bootable usb
+
+Personally, I only had one usb stick available, but ideally you'd have another to put Mint on. I'll cover this later, but after the Cloudready install, this usb stick can be wiped and used for the Mint install.
+
+When ready, follow these [instructions to boot Mint from a usb](https://community.linuxmint.com/tutorial/view/744).
+
+
+#####Setting up the BIOS
+
+Throughout the setup process you may come across a few recommended changes to the BIOS to get everything working. Might as well do these now. This HP machine has its own HP version of the BIOS, so some things may be specific to HP. I tried to keep things as general as possible, but the [archlinux wiki](https://wiki.archlinux.org/index.php/GRUB/EFI_examples) has detailed info for setting the UEFI mode for any BIOS.
```
Comment (mediocregopher) [inline _posts/2016-04-26-cloudready-linux-mint.markdown]: Shoutouts to arch linux for having such good docs that other distros don't even need their own

File: _posts/2016-04-26-cloudready-linux-mint.markdown [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+
+#####Partioning space for Mint
+
+Select "Something else" from the partition prompt. Now we can actually see what Cloudready has done to the drive.
+
+As the picture below depicts, Cloudready has made several partitions beyond just the original Windows partition.
+
+
+![Linux Mint partition menu after Cloudready](http://i.imgur.com/ZFkdpWF.png)
```
Comment (mediocregopher) [inline _posts/2016-04-26-cloudready-linux-mint.markdown]: more images for this repo

**Comment by mediocregopher** [comment]
Made some comments, I think one thing I felt was missing while I read through it was that there wasn't really an explanation for *why* you want to dual-boot chromeOS and linux mint. You explained that on the older laptop chromeOS would get slow, but that doesn't really apply to this new laptop which is the subject of the post. What does having mint on the laptop give you?

**Comment by likethemammal** [comment]
This post was about putting Mint and cloudready on *my* old machine, which would be estelle's new one. This one has nothing to do with the new razer blade i got. Was that not clear in the post?

**Comment by likethemammal** [comment]
K, @mediocregopher addressed your comments and re-worded the prologue a little to emphasize that dual-booting with Cloudready was to ease the transition for Estelle to this new comp

**Pull request merged**